include/libtorrent/alert_types.hpp

```diff
@@ -1398,7 +1398,7 @@ namespace libtorrent
 
 		TORRENT_DEFINE_ALERT(portmap_log_alert, 52)
 
-		static const int static_category = alert::port_mapping_notification;
+		static const int static_category = alert::port_mapping_log_notification;
```

Comment:
oh, right. this is the alert. would you mind mentioning in the documentation for this alert that it's in this category?